src/core/Select/Select.tsx

```diff
@@ -28,6 +28,14 @@ export type SelectOption<T> = {
    * Label of the item used in dropdown list and when selected.
    */
   label: string;
+  /**
+   * Sublabel of the item shown below the label when item size is 'large'.
+   */
+  sublabel?: React.ReactNode;
```

Comment:
tests

Reply:
Added

Reply:
Add test for the case, when size not passed, but has sublabel

Reply:
Approving, so you could merge when test is there